modules/dxp/apps/scim/scim-rest-impl/src/main/java/com/liferay/scim/rest/internal/manager/UserManagerImpl.java

```diff
@@ -386,11 +389,21 @@ public UsersGetResponse listUsersWithGET(
 					ExpressionNode expressionNode = (ExpressionNode)node;
 
 					if (expressionNode != null) {
-						if (expressionNode.getAttributeValue().contains("externalId")) {
+						if (expressionNode.getAttributeValue(
+							).contains(
```

Comment:
Hey @jgarcialr, please extract the `expressionNode.getAttributeValue()` to a variable or use `StringUtil#contains` to avoid this chaining here.

Reply:
got it, used colon as delimiter because all the values of scim resources are delimited by that character